rpc/types/src/bin.rs

```diff
@@ -91,15 +94,29 @@ define_request_and_response! {
     }
 }
 
+#[cfg(feature = "epee")]
+define_request_and_response! {
+    get_o_indexesbin,
+    cc73fe71162d564ffda8e549b79a350bca53c454 =>
+    core_rpc_server_commands_defs.h => 487..=510,
+    GetOutputIndexes,
+    #[derive(Copy)]
+    Request {
+        txid: [u8; 32],
+    },
```

Comment:
If we really want (memory) efficient parsing this (and other `[u8; N]` in `bin`) can be made into a `ByteArray`

Reply:
Can type optimizations be for a different PR? Also, the `cuprate_epee_encoding` blanket impls don't cover `Vec<ByteArray<N>>`.

Reply:
yeah sure, btw this `Vec<ByteArray<N>>` should always be this `ByteArrayVec<N>`